src/ajax/mobidetails.php

```diff
@@ -111,10 +111,14 @@
     $_SESSION['csrf_tokens']['mobidetails_confirm'] = md5(uniqid());
```

Comment:
Add documentation stating that flush() doesn't work; Suppress messages when ob_end_flush() fails by adding a '@'.